### PR TITLE
Adds USBConnectionEvent

### PR DESCRIPTION
**Pull request by rachelandrew**
Adds the missing interface from the WebUSB API.

Reviewer @jpmedley

### CODE REVIEW

**Review by jpmedley — CHANGES_REQUESTED** [review]
This looks good. Most comments are updates to syntax.

File: files/en-us/web/api/usbconnectionevent/device/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - device
+  - USBConnectionEvent
+---
+<div>{{securecontext_header}}{{DefaultAPISidebar("WebUSB API")}}</div>
+
+<p class="summary">The <strong><code>device</code></strong> read-only property of the {{domxref("USBConnectionEvent")}} interface returns a {{domxref("USBDevice")}} object representing the device being connected or disconnected.</p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="syntaxbox">let device = USBConnectionEvent.device;</pre>
```
[the remark this answers is not in the document]
Comment (rachelandrew) [reply files/en-us/web/api/usbconnectionevent/device/index.html:17]: The properties mentioned in the new syntax are still like this (in fact they use var...) https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#properties 

I queried it in chat and was reminded of https://github.com/mdn/content/issues/4476 which is not yet resolved. Therefore I didn't change these.

File: files/en-us/web/api/usbconnectionevent/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<h2 id="Properties">Properties</h2>
+
+<dl>
+  <dt>{{domxref("USBConnectionEvent.device")}}{{ReadOnlyInline}}</dt>
+  <dd>Returns a {{domxref("USBDevice")}} object representing the current device.</dd>
+</dl>
+
+<h2 id="Examples">Examples</h2>
+
+<p>In the following example listening for connect and disconnect events is used to add and remove the devices from the user interface of an application.</p>
```
[the remark this answers is not in the document]
Comment (rachelandrew) [reply files/en-us/web/api/usbconnectionevent/index.html:30]: well, whatever you wanted to do. Show a list of all devices or something. This is your own UI, your own webpage, not something generated by the browser (as in something like background fetch). We don't typically start writing stuff like that in these examples.

File: files/en-us/web/api/usbconnectionevent/device/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/usbconnectionevent/device/index.html:17]: I've changed the suggestion.

[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/usbconnectionevent/device/index.html:17]: I'll update the template.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**